@xen-orchestra/web/src/components/backups/panel/card-items/BackupJobTargetsSrItem.vue

```diff
@@ -1,16 +1,21 @@
 <template>
   <li>
-    <UiLink size="small" icon="fa:database" :href="`/#/srs/${sr.id}/general`">
+    <UiLink size="small" icon="fa:database" :href="`${xo5Route}#/srs/${sr.id}/general`">
       {{ sr.name_label }}
     </UiLink>
   </li>
 </template>
 
 <script setup lang="ts">
+import { useXoRoutes } from '@/remote-resources/use-xo-routes.ts'
 import UiLink from '@core/components/ui/link/UiLink.vue'
 import type { XoSr } from '@vates/types'
+import { computed } from 'vue'
 
 const { sr } = defineProps<{
   sr: XoSr
 }>()
+
+const { routes } = useXoRoutes()
+const xo5Route = computed(() => routes.value?.xo5 ?? '/')
```

Comment:
```suggestion
const xo5Route = computed(() => routes.value?.xo5 ?? '')
```
Should we remove the `/` ? @OlivierFL ? @ByScripts  ?

Reply:
I think that if `routes.value` is undefined, we should disable the link and add a log (because this shouldn't happen and indicates a problem somewhere)

Reply:
@J0ris-K @MathieuRA the composable has been updated [in this commit](https://github.com/vatesfr/xen-orchestra/pull/9247/commits/7a5bcc2dc41c7903f2b53a9c3d0fcfe582296afa#diff-d51c9cb24a9daac51ceb072958d6eb2061b2a51a7074567dd20a3274267c8360R1-R34) to ease the computation of the route value and log an error if the routes can’t be fetched.